src/guides/aaa_general/aaa_basics.mdx

```diff
@@ -6,37 +6,39 @@ route: /how-to-play-online-werewolf
+- Actions that gather intelligence (such as that of the Seer). A user of this action will receive a report in the morning revealing some information about their target, such as their role alignment, the players they visited (used their actions on) that night, or other auxilliary information. Each role page in this How to Play guide contains data about the kind of information that these intelligence roles can see.
+- Actions that protect against kills (such as that of the Protector). A target of this action is protected from attempted kills against them during the night on which it is used. In most cases, the kill action simply fails, though some protective actions cause other effects to happen if someone attempts to kill the protected player - for instance, the Huntsman will kill all players that attempt to attack their target, but will also die in the process. Some kill actions are able to bypass protection, such as that of the Militia.
+- Actions that revive players (such as that of the Reviver). A rare action, this allows a previously killed or lynched player to return to the game. In addition to regaining their night actions (unless already used up), night chats, and voting abilities, they may also carry information with them that they learned from the Graveyard chat.
+- Actions that block roles (such as that of the Direwolf). If a player is targeted by a successful roleblocking action, their own night action will not take place. These actions are resolved close to first in the order of night actions in order to block as many actions as possible. Some characters, such as the Vampire and roleblockers themselves, are immune to role blocks.
+- Actions that swap identities (such as that of the Shapeshifter). Targets of these actions swap their avatar and username assignments - this swap is not made known to anyone besides the targets, and each will appear as their new identities to the rest of the village. These swaps, however, do not exchange any other characteristics of the players, including their factions, night chats, abilities, and statuses. These actions are always performed last, just before kills and revives are reported to the village, which means that a swap cannot change the target of your night action, but that any intelligence reports you get might instead apply to a different identity after you get them.
 
-Note that all roles get reports on the situation before identity-swapping abilities, meaning that your information is possibly outdated by the time you receive it.
+Night actions are committed simultaneously at the end of the night and are resolved in a mostly deterministic way by the game engine. This order is detailed on the Night Action Order page.
```

Comment:
Would either link directly to the page or bring it back to this one

Reply:
How do I link pages in this engine? The H2P at present contains no links.

Reply:
Simple Markdown links _should_ suffice, so `[link](/url)`.

Reply:
Yep, that works! Thank you.